samcli/commands/check/resources/Pricing.py

```diff
@@ -0,0 +1,155 @@
+"""
+Pricing questions are asked here. Pricing is only done for
+Lambda Functions as of now. Data is stored in graph in
+Lambda function pricing object.
+"""
+
+from typing import List, Tuple
+import click
+
+from samcli.commands.check.resources.Graph import Graph
+from samcli.commands.check.resources.LambdaFunctionPricing import LambdaFunctionPricing
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class Pricing:
+    _graph: Graph
+    _max_num_requests: int
+    _min_memory_amount: int
+    _max_memory_amount: int
+    _max_duration: int
+
+    def __init__(self, graph: Graph) -> None:
+        """
+        Args:
+            graph (Graph): The graph object. This is where all of the data is stored
+        """
+        self._graph: Graph = graph
+        self._max_num_requests: int = 1000000000000000000000
+        self._min_memory_amount: int = 128
+        self._max_memory_amount: int = 10000
+        self._max_duration: int = 900000
```

Comment:
Why we have defaults for these values? Please make them clear that why we have these values as defaults.

Reply:
I will add comments explaining the meaning behind each one